package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby",
   "description": "React.js Static Site Generator",
-  "version": "0.6.2",
+  "version": "0.7.0",
```

Comment:
Agreed :-) the node.js project tags PRs with what kind of release is needed
e.g. semvar-minor which I think is a nice pattern.
On Nov 10, 2015 5:20 PM, "Tim Dorr" <notifications@github.com> wrote:

> In package.json
> <https://github.com/gatsbyjs/gatsby/pull/81#discussion_r44491305>:
>
> > @@ -1,7 +1,7 @@
> >  {
> >    "name": "gatsby",
> >    "description": "React.js Static Site Generator",
> > -  "version": "0.6.2",
> > +  "version": "0.7.0",
>
> I would recommend against including version changes in PRs. If this gets
> superseded by another version update, you'll end up with a merge conflict
> and it will be harder for the maintainer to merge. Just a heads up! [image:
> :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gatsbyjs/gatsby/pull/81/files#r44491305>.
>